stdlib/2and3/copy.pyi

```diff
@@ -0,0 +1,11 @@
+# Stubs for copy
+
+from typing import TypeVar
+
+_T = TypeVar('_T')
+
+# Note: deepcopy has other kwargs but they are only meant to be used internally.
```

Comment:
I feel like we should include these arguments anyway, since people may be using them and I'd rather not create false positive type errors for them.